worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/WorldEditPlugin.java

```diff
@@ -396,7 +397,9 @@ public void onDisable() {
         if (config != null) {
             config.unload();
         }
-        this.getServer().getScheduler().cancelTasks(this);
```

Comment:
This can just be fully removed, the server already calls it when disabling the plugin

Reply:
Gotcha!